### DISCUSSION TITLE
CUDA out of memory Anyone can solve this?

### DISCUSSION THREAD

**Discussion by wqf525** [deforum-art/sd-webui-text2video#15] [Q&A]
I use my laptop to get this extension, My gpu is 4060 140w laptop 8GB

```
CUDA out of memory. Tried to allocate 20.00 MiB (GPU 0; 8.00 GiB total capacity; 7.13 GiB already allocated; 0 bytes free; 7.27 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
```

I am confused with this , I use nvidia sim, my GPU is empty... anyone can do some help on this?

**Comment by zejacky** [comment]
There is a checkbox inside the extension gui "Low VRAM VAE". Maybe you tried already.

**Comment by Crimsonfart** [comment]
I have the same problem. Got a GTX 3060TI 8GB VRAM. The problem also occurs with 128x128, 5 frames, and low VRAM checks. Why could that be. I closed all programs in the background and have no problems with SD.

CUDA out of memory. Tried to allocate 12.00 MiB (GPU 0; 8.00 GiB total capacity; 7.19 GiB already allocated; 0 bytes free; 7.34 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

**Comment by kabachuha** [comment]
Added VAE halving, should use it way less now.

**Reply by kabachuha** [reply]
Ofc, any community optimizations are welcome if you have any ideas!

**Comment by Ragamuffin20** [comment]
so no ones found a solution?
